### PR TITLE
Microsoft feedback

### LINKED ISSUE #92: Language around sender constraints and implicit flow gives a misleading impression
In my opinion, this language (from 2.1.2):

> no viable method for sender-constraining exists to bind access tokens to a specific client (as recommended in Section 2.2) when the access tokens are issued in the authorization response

leads the reader to the wrong conclusion. I'm not sure it's even literally true.

It's true that the IETF has not standardized a sender-constraining mechanism for the implicit flow. However, this fact is a result of the priorities of IETF contributors more-so than a technical limitation. On Microsoft Entra, we do have a viable method for sender-constraining the implicit flow (since 2018). It's not technically complex to define such a mechanism. Taking the form serialization method defined in [Open ID Connect Core](https://openid.net/specs/openid-connect-core-1_0.html) Section 13.2, the DPoP proof can be passed directly to the authorization endpoint in a POST request, e.g. in a parameter "dpop". This is all it really takes to define a viable method for sender-constraining access tokens issued from the authorization endpoint.

My suggestion would be to drop the word "viable" which leads the reader to think about technical viability and replace it with the word "standardized":

> no **standardized** method for sender-constraining exists to bind access tokens to a specific client (as recommended in Section 2.2) when the access tokens are issued in the authorization response

This gives the correct impression - that the gap is around collaboration and interoperability.

### LINKED ISSUE #93: "4.2 Credential Leakage via Referer Headers" is out of date
Browsers largely mitigated the referer header threat in 2020 and 2021 when they updated the default Referrer Policy to "strict-origin-when-cross-origin". See https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referrer-Policy#browser_compatibility. The BCP makes no mention of this fact.

Oddly, the BCP makes mention of a security bug that is more than a decade old and hasn't existed in any supported software stack in nearly as long ("Chromium Issue 168213"). These browser versions are vulnerable to many other techniques - e.g. they predate Spectre and the [Post-Spectre Threat Model Re-Think](https://chromium.googlesource.com/chromium/src/+/master/docs/security/side-channel-threat-model.md) and TLS 1.0 deprecation.

I'd suggest moving the entirety of 4.2 to a separate section or appendix about historical attacks that are of limited relevance in contemporary software environments. If this is too large a change, 4.2 should at least explain the current context - that most forms of this attack are mitigated by default by the browser, and only same-origin attacks or attacks on ancient browser versions are relevant.

I'd suggest striking all mentions of Chromium Issue 168213 - this bug isn't notable enough.

### LINKED ISSUE #97: Mention response_mode=fragment in 4.1.2
Microsoft Entra supports response_mode=fragment from https://openid.net/specs/oauth-v2-multiple-response-types-1_0.html. Microsoft Entra's browser-based apps library (https://github.com/AzureAD/microsoft-authentication-library-for-js) uses this mode with the authorization code flow. The draft says:

> Additionally, implicit clients can be subject to a further kind of attack.

It would be nice to clarify that both implicit clients and non-implicit clients using providers that support response_mode=fragment can be subject to this attack.

### PR DESCRIPTION
**Pull request by danielfett**
Fixes #92

Fixes #93 

Fixes #97

### CODE REVIEW

**Review by will-bartlett — APPROVED** [review]

File: A04_attacks-and-mitigations.md [hunk 1]
```diff
@@ -211,8 +211,7 @@ from the authorization server's or the client's website, respectively. Most
 importantly, authorization codes or `state` values can be disclosed in
 this way. Although specified otherwise in [@RFC9110], Section 10.1.3,
 the same may happen to access tokens conveyed in URI fragments due to
-browser implementation issues, as illustrated by Chromium Issue 168213
-[@bug.chromium].
+browser implementation issues, as illustrated by a (now fixed) issue in the Chromium project [@bug.chromium].
```
Comment (will-bartlett) [inline A04_attacks-and-mitigations.md:214]: It's still my feeling that this bug is too small to mention. Chrome has had hundreds of security bugs in the past decade, but this document doesn't suggest (for example) that OAuth implementations take additional steps to add randomness to tokens to mitigate [CRIME](https://en.wikipedia.org/wiki/CRIME).

File: B_references.md [hunk 2]
```diff
@@ -53,7 +53,24 @@
   </front>
 </reference>
 
-
+<reference anchor="OpenID.MRT" target="https://openid.net/specs/oauth-v2-multiple-response-types-1_0.html">
```
Comment (selfissued) [inline B_references.md:56]:
```suggestion
<reference anchor="OAuth.Responses" target="https://openid.net/specs/oauth-v2-multiple-response-types-1_0.html">
```

File: A04_attacks-and-mitigations.md [hunk 3]
```diff
@@ -106,7 +106,7 @@ The attack described above works for the implicit grant as well. If
 the attacker is able to send the authorization response to an attacker-controlled URI, the attacker will directly get access to the fragment carrying the
 access token.
 
-Additionally, implicit clients can be subject to a further kind of
+Additionally, implicit grants (and also other grants when using `response_mode=fragment` as defined in [@OpenID.MRT]) can be subject to a further kind of
```
Comment (selfissued) [inline A04_attacks-and-mitigations.md:109]:
```suggestion
Additionally, implicit grants (and also other grants when using `response_mode=fragment` as defined in [@OAuth.Responses]) can be subject to a further kind of
```

Comment (selfissued) [reply A04_attacks-and-mitigations.md:109]: Please use the same reference name as in OpenID.Core

**Review by selfissued — APPROVED** [review]
Please accept my suggestion to change OpenID.MRT to OAuth.Responses.

**Pull request merged**